### PR TITLE
CONTRIBUTING.md explains rundata_test.go

### PR DESCRIPTION
**Pull request by liulk**

### CODE REVIEW

**Review by sachendras — DISMISSED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -117,18 +117,21 @@ For example:
 Within each test directory, `README.md` should document the test plan. The test
 name directory and the `*.go` files should be named after the test name as shown
 in the [project](https://github.com/orgs/openconfig/projects/2/views/1) item.
+
+Each test must also be accompanied by a `rundata_test.go` file that is
+automatically generated using the command `go run ./tools/addrundata --fix`. The
+rundata is for tracking the test result. If the rundata is outdated, the
+"Rundata Check" in the pull request check will fail.
```
Comment (dplore) [inline CONTRIBUTING.md:124]:
```suggestion
Each test must also be accompanied by a `rundata_test.go` file that is
automatically generated using the command `go run ./tools/addrundata --fix`. rundata_data
generates test results in junitxml format. If the rundata is outdated, the
"Rundata Check" in the pull request check will fail.
```

Would you mind adding link to a new a readme at tools/addrundata/readme.md with a quick explanation.  Others may find the test result output useful.

**Review by dplore — APPROVED** [review]
LGTM, added a suggestion for consideration

File: CONTRIBUTING.md (hunk 1 above)
Comment (liulk) [reply CONTRIBUTING.md]: Added tools/addrundata/README.md and moved detail there.

**Review by dplore — APPROVED** [review]
Very nice! Thank you.

**Pull request merged**